benchmark/README.md

```diff
@@ -55,6 +56,7 @@ They are defined as follows:
 * `certificates_synchronize_timeout`: The timeout configuration when requesting certificates from peers.
 * `payload_synchronize_timeout`: Timeout when has requested the payload for a certificate and is waiting to receive them.
 * `payload_availability_timeout`: The timeout configuration when for when we ask the other peers to discover who has the payload available for the dictated certificates.
+* `handler_certificate_deliver_timeout`: The timeout when waiting on the block synchronizer handler for a certificate fetched from peers to be delivered after submitted to core for processing & validation
```

Comment:
I think we need to either point to other documentation for a longer explanation, or describe what the waiting period is in this sentence. Something like "a timeout for checking the validity of a retrieved certificate, including a potential missing parent".

Also, ideally, I suspect we will grow to want an end-to-end timeout per asset (Certificate of block) rather than many intermediate ones.

Reply:
Agreed, I also believe that there are now too many timeouts here and there. I thin - and this is me not be Rust-oriented now - it would be great if we could have something like Go has with [context & timeouts](https://golang.cafe/blog/golang-context-with-timeout-example.html) which we could propagate through calls as a way to deal with this. In general I believe we can come up with something which would simplify this.